<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAprobacionesEntrada</class>
 <widget class="QDialog" name="dlgAprobacionesEntrada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aprobaciones</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Personas</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Cargo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtCargo"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="btPersona">
             <property name="text">
              <string>Pe&amp;rsona</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtPersona"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="btProvincia">
             <property name="text">
              <string>&amp;Provincia</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtProvincia"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Institución</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Tipo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtTipoInstitucion"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="btCasa">
             <property name="text">
              <string>Ca&amp;sa</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtCasa"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="btProvinciaInstitucion">
             <property name="text">
              <string>&amp;Provincia</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtProvinciaInstitucion"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Infos extra</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="WidgetExtraInfos" name="wdExtras" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Notas</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="WidgetNotas" name="wdNotas" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="twAprobaciones"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btAnadirAprobacion">
       <property name="text">
        <string>Aña&amp;dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarAprobacion">
       <property name="text">
        <string>Quitar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>Can&amp;celar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetExtraInfos</class>
   <extends>QWidget</extends>
   <header>widgets/widgetextrainfos.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetNotas</class>
   <extends>QWidget</extends>
   <header>widgets/widgetnotas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
